### PR TITLE
update to M1 L1 from rk

### PR DESCRIPTION
**Pull request by Robbiekruszynski**
reviewed M1 L1
changed a small portion of the wording 
no CSS / HTML updates

### CODE REVIEW

File: M1/L2/index.html [hunk 1]
```diff
@@ -60,15 +60,15 @@ <h3>The error message is your friend</h3>
 
     <p>This process is not necessarily intuitive and may take some time to learn. But after a while, you will develop a sense of the best words to use to search to get the best results. You'll also start to recognize websites that seem well-structured and informative.</p>
 
-    <p>Most despairingly, typically solving one problem will lead to another problem surfacing. Take a deep breath and repeat the process on <i>that</i> problem. But be sure to remember the issues as they arise. The knowledge you gain from solving one problem on a project stands to benefit any future problems.</p>
```
Comment (ConsenSys-Academy) [inline M1/L2/index.html:63]: I know it's a bit wordy, but I think it's important to clarify it's not all the time. Would you mind removing this change?

File: M1/L1/index.html [hunk 2]
```diff
@@ -1,56 +1,116 @@
...
+      encourage you to flip through them. Along with introductory material,
+      we’ve curated tips and tricks for optimizing and improving your use of
+      these tools. After all, if you're staring at a code editor for hours,
+      shouldn't it be tricked out in the best way possible? Who knows, what you
+      learn here could make your developer experience not only more efficient,
+      but fun!
+    </p>
 
-  <p>Luckily, to get comfortable you don't need to know that many things. It's just that they don't typically get explained together all at once.</p>
+    <h2>A Few Things</h2>
```
Comment (ConsenSys-Academy) [inline M1/L1/index.html:70]: I'm cool with a new header here, but perhaps more descriptive?

**Review by ConsenSys-Academy — CHANGES_REQUESTED** [review]
Hey! Most looks good, just a few notes here, do you have the formatting header changes for L2 as well?

I apologize you have to do this for each section, ideally we'd have a header / footer template, but we're not quite there yet.

File: M1/L1/index.html (hunk 2 above)
Comment (Robbiekruszynski) [reply M1/L1/index.html:70]: I think <h2>A Few Things</h2> was pulled from the original looks to exists in 

basic-training/M1/L1/index.html main branch

File: M1/L2/index.html (hunk 1 above)
Comment (Robbiekruszynski) [reply M1/L2/index.html:63]: added back the original text

**Review by ConsenSys-Academy — APPROVED** [review]
